roles/oauth/templates/upstream.conf.j2

```diff
@@ -9,10 +9,13 @@ server {
   listen 1443;
   server_name {{ oauth_domain_name }};
 
+  # Somthing that arrives as https://oauth-{{name}}.dev.lcip.org/
+
   location / {
     proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
     proxy_set_header Host $http_host;
     proxy_redirect off;
+    rewrite ^/(.*)$ /auth/$1 break;
     proxy_pass http://upstream_oauth_server;
   }
```

Comment:
For fxa-dev, all the hostnames already resolve to the same "host". It's nginx rules that decide where things go, and the above send "oauth" requets to the auth daemon. I think.